app/src/main/java/com/example/arc_exapmle/NoteRepository.kt

```diff
@@ -39,43 +39,48 @@ class NoteRepository(application: Application) {
 
     }
 
-    fun getAllNotes(): LiveData<List<NoteEntity>?> {
+    fun getAllNotes(): LiveData<List<NoteEntity>> {
+
         return allNotes
     }
 
-    private class InsertNoteAsyncTask(noteDaoNew: NoteDao) : AsyncTask<NoteEntity, Unit, Unit>() {
+    private class InsertNoteAsyncTask(noteDaoNew: NoteDao) : AsyncTask<NoteUI, Unit, Unit>() {
```

Comment:
NoteUi ro behtare to ViewModel tabdil be NoteEntity koni
baad befresti baraye repo

Reply:
manam hamin kar ro anjam dadam diige 
albate hala ye taghirati behesh dadam 
bebinin ishalla hal shode bashe in masale ham